File: code/backend/DarkDeeds.TelegramClientApp.Infrastructure/Communication/TaskServiceApp/Dto/TaskDto.cs
using System;
using DarkDeeds.TelegramClientApp.Infrastructure.Communication.TaskServiceApp.Enums;

namespace DarkDeeds.TelegramClientApp.Infrastructure.Communication.TaskServiceApp.Dto
{
    public class TaskDto
    {
        public int Id { get; set; }
        public DateTime? Date { get; set; }
        public int? Time { get; set; }
        public string Title { get; set; }
        public int Order { get; set; }
        public int ClientId { get; set; }
        public bool Completed { get; set; }
        public bool IsProbable { get; set; }
        public bool Deleted { get; set; }
        public TaskTypeEnum Type { get; set; }
        public int Version { get; set; }
        public string Uid { get; set; }
    }
}

File: code/frontend4/src/tasks/models/TaskDto.ts
import { TaskTypeEnum } from './TaskTypeEnum'

export interface TaskDto {
    uid: string
    title: string
    date: Date | null
    order: number
    completed: boolean
    deleted: boolean
    type: TaskTypeEnum
    isProbable: boolean
    version: number
    time: number | null
}
